packages/catalog-search/src/SearchBox.jsx

```diff
@@ -54,40 +65,65 @@ export const SearchBoxBase = ({
     }
   };
 
+  const handleSuggestionSubmit = (query, hit) => {
```

Comment:
the understanding i got from this function is that when we wont provide the override then clicking on the `SearchSuggestionItem` wont be affected on learner portal, is that right?


and about the click on view all results i have a concern which is mentioned in another comment.

Reply:
this method is provided to the suggestion items and is only invoked when `disableSuggestionRedirect` is true. Instead of redirecting the user to the url (in the learner portal's case, the enterprise course page), it defaults to whatever functionality was provided through this method.

Reply:
oky cool, thanks @alex-sheehan-edx @binodpant